### PR TITLE
Added 'clear all filters' to learner search

### LINKED ISSUE #849: Add clear all button to search results

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

closes #849 

#### What's this PR do?

This just adds a little searchkit component that provides a 'clear all filters' which removes all the filters which are currently applied.

### CODE REVIEW

File: static/js/components/LearnerSearch.js [hunk 1]
```diff
@@ -102,6 +103,7 @@ export default class LearnerSearch extends SearchkitComponent {
                 </Cell>
```
Comment (gsidebo) [inline static/js/components/LearnerSearch.js]: this is totally not part of this PR, but 10 lines above this, the "Email Selected" button is given `id="download csv"`. think that's pretty obviously a typo. since youre already in this code right now, would you mind changing that as part of this PR or some other learner search front end PR?

**Comment by gsidebo** [comment]
👍

File: static/js/components/LearnerSearch.js (hunk 1 above)
Comment (aliceriot) [inline static/js/components/LearnerSearch.js]: lol yeah that sounds like a typo!

Comment (aliceriot) [inline static/js/components/LearnerSearch.js]: I'll change it here.
